Sources/MobileCoinClient.swift

```diff
@@ -189,6 +189,40 @@ public final class MobileCoinClient {
         ).requiresDefragmentation(toSendAmount: amount, feeLevel: feeLevel, completion: completion)
     }
 
+    public func createSignedContingentInput(
+        recipient: PublicAddress,
+        amountToSend: Amount,
+        amountToReceive: Amount,
+        completion: @escaping (
+            Result<SignedContingentInput, SignedContingentInputCreationError>
+        ) -> Void
+    ) {
+        guard let rngSeed = defaultRng.generateRngSeed() else {
+            completion(.failure(
+                SignedContingentInputCreationError.invalidInput("Couldn't create 32byte RNG seed")))
+            return
+        }
+        Account.SCIOperations(
```

Comment:
I doesn't make sense to init this struct and not store it anywhere. the init part should be moved up into the account init then when you need to call an "SCIOperation" you call it on that variable

Reply:
although I see this is an existing pattern so we can keep it the way you have it now.